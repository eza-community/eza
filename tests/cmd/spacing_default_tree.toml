bin.name = "eza"
args = "tests/itest --tree"
