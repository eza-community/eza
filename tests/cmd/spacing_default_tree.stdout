tests/itest
├── a
├── b
├── c
├── d
├── dir-symlink -> vagrant/debug
├── e
├── exa
│   ├── file.c -> djihisudjuhfius
│   └── sssssssssssssssssssssssssggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggsssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss
│       └── Makefile
├── f
├── g
├── h
├── i
├── image.jpg.img.c.rs.log.png
├── index.svg
├── j
├── k
├── l
├── m
├── n
├── o
├── p
├── q
└── vagrant
    ├── debug
    │   ├── a
    │   ├── symlink -> a
    │   └── symlink-broken -> ./b
    ├── dev
    │   └── main.bf
    └── log
        ├── file.png
        └── run
            ├── run.log.text
            └── sps.log.text
